css/styles.css

```diff
@@ -0,0 +1,399 @@
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    text-transform: uppercase;
+    color: #FFFFFF;
+}
+
+/*********************************************************/
+
+.tagline    {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: flex-start;
+    padding: 0px;
+}
+
+.main-tagline   {
+    width: 750px;
+    height: 240px;
+    font-size: 72px;
+    line-height: 80px;
+    text-transform: capitalize;
+    text-align: left;
+    text-transform: capitalize;
+}
+
+.main-tagline>p     {
+    font-family: Poppins;
+    font-size: 20px;
+    line-height: 30px;
+}
+
+/*.numbers    {
+
+} */
+.numbers-item   { 
+    font-family: Bakbak One, sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 44px;
+    line-height: 56px;
+    display: flex;
+    align-items: flex-end;
+    text-transform: capitalize;
+    color: #FFFFFF;
+    text-shadow: 0px 4px 13px #ffffff66;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+}
+
+.numbers-text       {
+    font-family: Poppins;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 26px;
+    display: flex;
+    align-items: flex-end;
+    color: #C2C3C5;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+}
+
+
+.connect-wallet {
+    color: #141b22;
+    font-family: Bakbak One, sans-serif;
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 24px;
+    letter-spacing: 0%;
+    text-align: left;
+    text-transform: uppercase;
+}
+
+.carusel-item>h3 {
+    font-size: 20px;
+    line-height: 28px;
+    text-transform: capitalize;
+}
+
+.about-us-hdr>h2 {
+    font-size: 44px;
+    line-height: 127%;
+    text-transform: capitalize;
+    text-align: center;
+    color: #fff;
+}
+
+.about-us-hdr>p {
+    font-size: 18px;
+    line-height: 28px;
+    text-align: center;
+    color: #C2C3C5;
+    flex: none;
+    order: 1;
+    align-self: stretch;
+    flex-grow: 0;
+}
+
+.list-item-abts>h3 {
+    font-size: 24px;
+    line-height: 30px;
+    text-transform: capitalize;
+}
+
+.list-item-abts>p {
+    font-size: 18px;
+    line-height: 28px;
+    color: #C2C3C5;
+}
+
+.readmore-about-us {
+    font-family: "Bakbak One", sans-serif;
+    font-weight: 400;
+    font-size: 44px;
+    line-height: 127%;
+    text-transform: capitalize;
+    text-align: center;
+    color: #fff;
+}
+
+.how-to-work-hdr {
+    font-size: 44px;
+    line-height: 56px;
+    text-align: center;
+    text-transform: capitalize;
+}
+
+.step-how-to {
+    list-style: none;
+}
+
+.step-how-to, dt {
+    font-family: "Bakbak One", sans-serif;
+    font-weight: 400;
+    font-size: 22px;
+    line-height: 136%;
+    text-transform: capitalize;
+    text-align: center;
+}
+
+.step-how-to, dd {
+    font-family: "Bakbak One", sans-serif;
+    font-weight: 400;
+    font-size: 24px;
+    line-height: 133%;
+    text-transform: capitalize;
+    text-align: center;
+    color: #fff;
+}
+
+.r-map-hdr {
```

Comment:
I would suggest naming the classes more fully. Perhaps `r` here meant `right` and `hdr` means `header`?